src/jinjanator/cli.py

```diff
@@ -304,7 +308,7 @@ def render_command(
 
     # We always expect a file;
     # unless the user wants 'env', and there's no input file provided.
-    if args.format == "env":
+    if args.format == "env" and args.data is None:
```

Comment:
It appears that Ruff must have made this change; it took a few minutes for me to confirm that it provides the same behavior. I am surprised it was able to make a complex change like this!